src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -40,6 +40,10 @@
     <f:textbox/>
   </f:entry>
 
+  <f:entry field="slaveConnectTimeoutStr" title="${%Timeout in seconds for slave 🡒 jenkins conn.}">
```

Comment:
Can you write `Jenkins connection`, for a better description
is there a bad char 🡒  ?